great_expectations/expectations/core/expect_column_max_to_be_between.py

```diff
@@ -104,6 +115,30 @@ class ExpectColumnMaxToBeBetween(ColumnExpectation):
         "profiler_config",
     )
 
+    max_range_estimator_parameter_builder_config: ParameterBuilderConfig = (
+        ParameterBuilderConfig(
```

Comment:
Just confirming: this is the pattern going forward for self-initializing Expectations right? I remember you mentioned it during our Standup today

Reply:
@Shinnnyshinshin This pattern will make things easier; however, the more verbose pattern is available as well.  I think there are four ways to pick from as situation calls for:
* Using `ParameterBuilderConfig` (as in above)
* Using `dict` version of the above
* Using `ParameterBuilderConfig.to_dict()` -- if starting with `ParameterBuilderConfig` and wanting a plain dictionary.
* Using the specific `ParameterBuilder` object and methods on it and/or `ParameterBuilder.config()` and methods on it and/or `ParameterBuilder.to_json_dict()` and keys from it.

Any and all these are supported in any combination.  For Self-Initializing Expectations, I would use the first or the fourth approaches.

Thanks!